tari-project/tari-dan
# feat: claim free testnet coins

Description
---
This PR provides functionality for airdropping tari on the DAN layer, thus avoiding to go through claiming burn fees.

Motivation and Context
---
Going through claim of burn fees is a tedious and time consuming process, as one has to burn fees on the base layer and then claiming it, while keeping track of all the cryptographic required data. Instead, we simplify the process to use a test tari airdrop command. 

How Has This Been Tested?
---
Manually

What process can a PR reviewer use to test or verify this change?
---
1. Start a validator node: 

    `$ cargo run --release --bin tari_validator_node -- -b . --network igor`

Then register it (either manually or through web ui).

2. Start wallet deamon:
    
    `$ cargo run --bin tari_dan_wallet_daemon -- -b . --network igor`

3. Create a new wallet account

    `$ cargo run --bin tari_dan_wallet_cli -- accounts create --account-name jorge_ant`

4. Invoke airdrop command

    `$ cargo run --bin tari_dan_wallet_cli -- accounts faucet --account-name jorge_ant --amount 10000000 --fee 1000`

5. Check balances of account

    `$ cargo run --bin tari_dan_wallet_cli -- accounts get-balances jorge_ant`





Breaking Changes
---
